crates/air-lib/test-utils/Cargo.toml

```diff
@@ -20,4 +20,6 @@ avm-server = { path = "../../../avm/server" }
 marine-rs-sdk = "0.6.15"
 
 fstrings = "0.2.3"
+object-pool = "0.5"
+once_cell = "1.10"
```

Comment:
Suggest to specify patch versions, we've experiences problems with accidentally breaking changes in patch versions of several crates.